apm-dist/pom.xml

```diff
@@ -34,6 +34,17 @@
             <activation>
                 <activeByDefault>true</activeByDefault>
             </activation>
+            <dependencyManagement>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.apache.skywalking</groupId>
+                        <artifactId>oap-server-bom</artifactId>
+                        <version>${project.version}</version>
+                        <scope>import</scope>
+                        <type>pom</type>
+                    </dependency>
+                </dependencies>
+            </dependencyManagement>
```

Comment:
Why do we need to add this?

Reply:
module apache-skywalking-apm  without  dependencyManagement can not handle dependency conflict and version management

Reply:
oap-server has that, and dist itself is just packaging, it should not include any new dependency AFAIK.